README.md

```diff
@@ -40,7 +40,9 @@ Start with these names for your branch depending on what your branch includes.
   - 'dependencies/'
   - 'documentation/'
   - 'enhancement/'
-  - 'feature/'
+  - feat/single-rights/issue-number
+  - feat/api-delegation/issue-number
+  - feat/general/issue-number       (When it's not related to any features)
```

Comment:
Should these also be annoted with ''?

Reply:
Hmm I don't know. Is the annotation necessary? I removed them on the other just now as a proposed solution

Reply:
Idk, I just noticed the difference 😅
I checked how it looks when "rendered" now and it looks good I think